ios/Classes/PdftronFlutterPlugin.h

```diff
@@ -143,19 +143,29 @@ static NSString * const PTAnnotationFlagPrintKey = @"print";
 static NSString * const PTAnnotationFlagReadOnlyKey = @"readOnly";
 static NSString * const PTAnnotationFlagToggleNoViewKey = @"toggleNoView";
 
+typedef enum {
+    exportAnnotationId = 0,
```

Comment:
`exportAnnotationId` used to start at 1, and now it's 0. Is this OK?

Reply:
Well, as long as they match on Native and Dart, it is OK. In fact, changing the numbers on iOS or Android native is easy, while Dart does not allow assignment, so the only way was to use the index, which starts from 0.